docs/cli.md

```diff
@@ -429,16 +516,24 @@ See [Datasets](/services/data/dataset/#datasets) for more information.
 
 ##### Using the `filter` argument
```

Comment:
I think this should also go to dataset section. And you can link to there from data without duplicating.

Reply:
it's used for both but I will move to dataset section